sniffles/feature.py

```diff
@@ -359,27 +418,73 @@ def parseLine(self, line=None):
                 return None
             if mypairs['type'].lower() == 'feature':
                 myfeature = Feature(name, lower_bound, upper_bound,
-                                    complexity_prob, myambiguity_list)
+                                    complexity_prob, ambiguity_list)
             elif mypairs['type'].lower() == 'content':
                 myfeature = ContentFeature(name, regex, complexity_prob, len)
             elif mypairs['type'].lower() == 'ip':
                 myfeature = IPFeature(name, version, complexity_prob)
             elif mypairs['type'].lower() == 'protocol':
                 myfeature = ProtocolFeature(name, proto_list, complexity_prob,
-                                            myambiguity_list)
+                                            ambiguity_list)
             else:
                 print("Unrecognized feature type.", line)
                 return None
             self.features.append(myfeature)
 
+    def tokenizeAmbiguityList(self, list):
+        notTrim = list[1:-1]
```

Comment:
notTrim is not a very descriptive name here.  listAsString might be better, or unparsedList.